resources/views/home.blade.php

```diff
@@ -17,9 +17,23 @@
             </p>
         </div>    
     @endif
+    
     <div class="mb-2">
-        <span>Your FTE: <b class="{{ auth()->user()->ftes['main'] < 1 ? 'text-danger' : 'text-success' }}">{{auth()->user()->ftes['main'] }}</b> </span>
+        <span>Your FTE: <b class="{{ auth()->user()->ftes['main'] < 1 ? 'text-danger' : 'text-success' }}">{{ auth()->user()->ftes['main'] }}</b></span>
     </div>
+
+    <form method="POST" action="{{ route('storeDropdownValue') }}">
+        @csrf
+        <div class="dropdown">
+            <select class="btn bg-light" name="selectedValue" onchange="this.form.submit()">
+                @foreach($centre as $location)
+                    <option value="{{ $location->centre_name }}">{{ $location->centre_name }}</option>
+                @endforeach
```

Comment:
Can we write like this?
```suggestion
                @foreach($centres as $centre)
                    <option value="{{ $centre->centre_name }}">{{ $centre->centre_name }}</option>
                @endforeach
```